linux_os/guide/system/accounts/accounts-pam/package_pam_pwquality_installed/rule.yml

```diff
@@ -28,9 +28,9 @@ references:
 ocil_clause: 'the package is not installed'
 
 {{% if 'ubuntu' not in product %}}
-{{{ complete_ocil_entry_package(package="libpwquality") }}}
+{{{ ocil_package(package="libpwquality") }}}
```

Comment:
It looks like you will need the `ocil` key here.

Reply:
that might be it

Reply:
if it's inline then it would need the single quotes. But multiline doesn't require it